frontend/src/app/dashboard/dashboard.component.ts

```diff
@@ -153,6 +156,9 @@ export class DashboardComponent implements OnInit {
     this.transactions$ = this.stateService.transactions$
       .pipe(
         scan((acc, tx) => {
+          if (acc.find((t) => t.txid == tx.txid)) {
```

Comment:
What does this fix again? Was there a problem where transaction IDs got duplicated in the latest transactions list? Why would there be duplicate txids?

Reply:
If you have less than 6 txs in that list and switch to a network with the backend offline, the txs stay there. When switching back, the txs were appended again. This was also for the "recent blocks".

Reply:
1st one of https://github.com/mempool/mempool/pull/1165#issuecomment-1017808054